source/geometries/Next100EnergyPlane.cc

```diff
@@ -238,7 +237,7 @@ namespace nexus {
 			  "EP_COPPER_PLATE");
 
     G4double stand_out_length =
-      sapphire_window_thickn_ + optical_pad_thickn_ + pmt_stand_out_;
+      sapphire_window_thickn_ + tpb_thickn_ + optical_pad_thickn_ + pmt_stand_out_;
```

Comment:
I think that the TPB thickness should not be added, since the TPB is placed inside the sapphire window, therefore its thickness is included in that of the window.

Reply:
I have added `tbp_thickn_` into the sapphire window volume, then this value must be added here also. In the previous implementation the `tpb_thickn_` is not added to the sapphire volume, I guess because it is negligible.

Reply:
Yes, it is irrelevant. Either implementation works.

Reply:
![Screenshot 2021-07-26 at 11 30 17](https://user-images.githubusercontent.com/8114436/126967029-2e9782ab-e830-4ead-9612-233a7704b6c2.png)

However, the placement of TPB doesn't seem right: a geantino coming from the inside of the chamber should see the TPB first, not the sapphire window.

Reply:
I think that this line:

`G4double tpb_posz = - sapphire_window_thickn_/2. + tpb_thickn_/2.;`

should rather be

`G4double tpb_posz = - (sapphire_window_thickn_+ tpb_thickn_)/2. - tpb_thickn_/2.;`

, which allows one to see the steps logically.

Reply:
I think that 

`G4double tpb_posz = - sapphire_window_thickn_/2. + tpb_thickn_/2.;`

should be replaced with

`G4double tpb_posz = - sapphire_window_thickn_/2. ;`, 

although I think that writing

`G4double tpb_posz = - (sapphire_window_thickn_ + tpb_thickn_)/2.  - tpb_thickn_/2.`

is clearer.

Reply:
Right, I forgot to add the `tpb_thickn_` to the translation in the sapphire volume.